src/errors/ApiError.ts

```diff
@@ -42,7 +32,7 @@ export default class ApiError extends Error {
    *
    * @public ✓ This method is part of the public API
    */
-  public getField(): string {
+  public getField(): string | undefined {
```

Comment:
In our internal codebase we use the types below. I would love to use these in here too.

```
declare type Maybe<T> = T | undefined;
declare type Nullable<T> = T | null;
```

Reply:
Cool.